German/main/FileExplorer.apk/res/values-de/strings.xml

```diff
@@ -173,14 +173,14 @@
   <string name="ftp_username_hint">Benutzername</string>
   <string name="gbk_summary">\"Vereinfachtes Chinesisch\" Windows Standardcodierung</string>
   <string name="gbk_title">GBK</string>
-  <string name="get_cloud_space">500GB hinzufügen</string>
+  <string name="get_cloud_space">Add 500GB</string>
```

Comment:
@vivanco-vivanco  Ist es absicht, dass hier bereits übersetze Strings wieder in das englische Equivalent überschrieben werden? (siehe auch Zeilen weiter unten)